watch_service.go

```diff
@@ -62,20 +62,20 @@ func (ws *watchService) watchFile(ctx context.Context, projectName, repoName, la
 	v := &url.Values{}
 	if query != nil && query.Type == JSONPath {
 		if err := setJSONPaths(v, query.Path, query.Expressions); err != nil {
-			watchResult <- &WatchResult{Err: err}
-			return watchResult
+			return &WatchResult{Err: err}
 		}
 	}
 	u += encodeValues(v)
-	ws.watchRequest(ctx, watchResult, u, lastKnownRevision, timeout)
-	return watchResult
-}
 
-func (ws *watchService) watchRepo(ctx context.Context,
-	projectName, repoName, lastKnownRevision, pathPattern string, timeout time.Duration) <-chan *WatchResult {
+	return ws.watchRequest(ctx, u, lastKnownRevision, timeout)
+}
 
-	// initialize watch result channel
-	watchResult := make(chan *WatchResult, 1)
+func (ws *watchService) watchRepo(
+	ctx context.Context,
+	projectName, repoName, lastKnownRevision,
+	pathPattern string,
```

Comment:
Could it better to append this line to the above line?

Reply:
Could you describe it in detail? I don't know what you mean @minwoox

Reply:
After I saw https://github.com/line/centraldogma-go/pull/9/files#diff-b290524cc65854c13e76a812764193beR46
I thought it could be better if the parameters and the type is at the same line which is
`projectName, repoName, lastKnownRevision, pathPattern string,`.
But if it's awkward, just ignore the comment. :)

Reply:
So let's keep it like moment ^_^